crates/bevy_transform/src/components/parent.rs

```diff
@@ -22,7 +22,11 @@ impl FromWorld for Parent {
 
 impl MapEntities for Parent {
     fn map_entities(&mut self, entity_map: &EntityMap) -> Result<(), MapEntitiesError> {
-        self.0 = entity_map.get(self.0)?;
+        // Parent of an entity in the new world can be in outside world, in which case it
+        // should not be mapped.
+        if let Ok(mapped_entity) = entity_map.get(self.0) {
+            self.0 = mapped_entity;
+        }
```

Comment:
Why is this special cased compared to other components that map entities?
Maybe we could always avoid unwrapping at the call sites of `map_entities` and log errors instead?